src/js/renderers.js

```diff
@@ -0,0 +1,49 @@
+var utils = require('./utils');
+
+var gene = function(config) {
```

Comment:
I'm not sure if 'gene' is the best name for this. What about something like 'gene_row'/'geneRow' ?